### DISCUSSION TITLE
Como apagar um repositório do github pelo terminal do ubuntu?

### DISCUSSION THREAD

**Discussion by rafgam10** [frontendbr/forum#2337] [Perguntas e Respostas]
PS:Tô falando de linha de codigo para apagar, ele do github.

**Comment by igorlvicente** [comment]
Usando o GitHub CLI: https://cli.github.com/manual/gh_repo_delete
`gh repo delete`

**Reply by rafgam10** [reply]
muito obg igor vc me ajudou muito!!! tem algum video que vc me recomenda para pesquisa melhor os comandos?

**Reply by igorlvicente** [reply]
O que eu recomendo fazer é ler a documentação bem rapidamente para ter uma ideia do que é possível ser feito e quais são os termos usados.
Depois disso, preste atenção no que você normalmente faz na interface do GitHub e procure os equivalentes nos comandos.
Não precisa decorar todos os comandos, só os que você normalmente usará.
Anote os comandos que você for usando em papel também pra criar uma _cheatsheet_ que está sempre à mão.

**Reply by rafgam10** [reply]
ok muito obg igor.
